pkg/client/deployspec_test.go

```diff
@@ -90,42 +90,3 @@ func Test_buildDeploySpecQueries(t *testing.T) {
 		})
 	}
 }
-
-func TestAuroraDeploySpec_Value(t *testing.T) {
```

Comment:
Must check if this test is required.

Reply:
I don't think so, it's covered by pkg/deploymentspec/deployspec_test.go